### PR TITLE
make README fragment syntactically correct

### PR DESCRIPTION
**Pull request by acjay**
Not sure if this sends the same message you originally had in mind, but I edited the fragment on my own until I got it to compile.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,23 +17,23 @@ libraryDependencies += "com.github.fommil" %% "spray-json-shapeless" % "1.2.0"
...
   Faz(Some("meh")).toJson        // """{"o":"meh"}"""
   Faz(None).toJson               // """{}"""
-  Foo("foo"): SimpleTrait.toJson // """{"type":"Foo","s":"foo"}"""
-  Bar(): SimpleTrait.toJson      // """{"type":"Bar"}"""
-  Baz: SimpleTrait.toJson        // """{"type":"Baz"}"""
-  Fuzz: SimpleTrait.toJson       // """{"type":"Fuzz"}"""
+  (Foo("foo"): SimpleTrait).toJson // """{"type":"Foo","s":"foo"}"""
+  (Bar(): SimpleTrait).toJson      // """{"type":"Bar"}"""
+  (Baz: SimpleTrait).toJson        // """{"type":"Baz"}"""
+  (Faz(None): SimpleTrait).toJson       // """{"type":"Fuzz"}"""
```
Comment (fommil) [inline README.md]: typo in comments

**Comment by fommil** [comment]
cool thanks. I've never heard of `faz` before.

**Comment by acjay** [comment]
I think you invented it :)

I guess maybe the one under `domain` should have been `Fuzz`? I just assumed the first appearance was the right one.

**Comment by fommil** [comment]
oh, so I did :smile: I can't remember doing that.

**Pull request merged**